completions/quick-lint-js.ps1

```diff
@@ -0,0 +1,16 @@
+Register-ArgumentCompleter -CommandName quick-lint-js -Native -ScriptBlock {
+    param ($wordToComplete)
+
+    $opts = @('--help',
+              '--version',
+              '--lsp-server',
+              '--config-file=',
+              '--stdin',
+              '--exit-fail-on=',
+              '--output-format=',
+              '--diagnostic-hyperlinks=',
+              '--vim-file-bufnr='
+            )
+
+    $opts.Where({$_ | Select-String $wordToComplete})
```

Comment:
Would this accidentally match `el` -> `--help`?

Reply:
It did. This has been fixed in e8a766e58a60f0414f66d7653913950c0e1842e2.